src/braintree/initBraintreePayPalButtons.ts

```diff
@@ -0,0 +1,310 @@
+        },
+    });
+    batchRequests.push({
+        apiType: 'setTaxes',
+        payload: {},
+    });
+    
+    let resp = await batchRequest(batchRequests);
+    /* istanbul ignore next */
+    if (!resp.success) {
+        displayError('There was an unknown error while processing your payment.', 'payment_gateway', 'unknown_error');
+        return;
+    }
+
+    resp = await addPayment({
+        gateway_public_id: payment.properties.public_id,
+        token: result.nonce,
+    });
+    /* istanbul ignore next */
+    if (!resp.success) {
+        displayError('There was an unknown error while processing your payment.', 'payment_gateway', 'unknown_error');
+        return;
+    }
+
+    orderProcessing();
+};
+
+const handleShippingChange = (
+    paypalCheckout: IBraintreePaypalCheckoutInstance
+) => async (
+    data: OnShippingChangeData,
+    actions: OnShippingChangeActions
+): Promise<void> => {
+    /* istanbul ignore next */
+    if (!data.shipping_address) {
+        return actions.reject();
+    }
+    
+    const currency = getCurrency().iso_code;
+    const resp = await estimateShippingLines({
+        city: data.shipping_address.city,
+        country_code: data.shipping_address.country_code,
+        province_code: data.shipping_address.state,
+        postal_code: data.shipping_address.postal_code,
+    } as unknown as IAddress);
+    /* istanbul ignore next */
+    if (!resp.success) {
+        actions.reject();
+        return;
+    }
+
+    let hasSelected = false;
+    const amountDue = getTotals().totalAmountDue;
+    const respData = (resp.response as IApiSuccessResponse).data as IEstimateShippingLinesResponse;
+    const serverSelectedShippingId = respData.application_state.shipping.selected_shipping?.id;
+    const options = respData.application_state.shipping.available_shipping_lines.map(o => {
+        hasSelected ||= serverSelectedShippingId === o.id;
+        return {
+            id: o.id, 
+            label: o.description,
+            selected: serverSelectedShippingId === o.id,
+            amount: {
+                value: (o.amount / 100).toFixed(2),
+                currency,
+            },
+            type: 'SHIPPING' as const,
+        };
+    });
+
+    if (!hasSelected && options && options.length) {
+        options[0].selected = true;
+    }
+
+    await paypalCheckout.updatePayment({
+        paymentId: data.paymentID as string,
+        currency,
+        amount: amountDue / 100,
+        shippingOptions: options,
+    });
+
+    actions.resolve();
+};
+
+async function makeButtons(payment: IExpressPayBraintreePaypal, paypalCheckout: IBraintreePaypalCheckoutInstance): Promise<void> {
+    const paypal = getPaypalNameSpace();
+    let enableSection = false;
+
+    const paypalButtonDiv = document.createElement('div');
+    const paypalButtonDivId = 'ppcp-paypal-express-payment-button';
+    paypalButtonDiv.id = paypalButtonDivId;
+    paypalButtonDiv.dataset.testid = paypalButtonDivId;
```

Comment:
Can you add the class name `express-payment` to both the paypal and paylater buttons so they match the existing express pay buttons. This was recently an issue with the ppcp buttons: https://github.com/bold-commerce/checkout-express-pay-library/pull/34

Reply:
Fixed